src/7-to-8/summary.rst

```diff
@@ -227,116 +124,20 @@ Optional and Expected Task Outputs
-task`.
-
-A task can finish with or without completing optional outputs, on the other
-hand. The primary use for optional outputs is alternate path branching in the
-graph.
-
-If there is nothing left to do, but incomplete tasks are present, the scheduler
-will conclude that the workflow did not run to completion as expected and will
-:term:`stall` rather than shut down.
-
-
-Window on the Workflow
-----------------------
-
-.. seealso::
-
-   * User Guide :ref:`n-window`
-
-
-.. image:: ../img/n-window.png
-   :align: center
-
-The Cylc UI can't show "all the tasks" at once because the graph may be huge,
-or even infinite in extent in cycling systems. The Cylc 8 UI shows:
-
-- Current **active tasks** (submitted, running) plus tasks waiting on scheduler
-  constraints (queues, runahead limit, clock triggers) and external triggers
-
-- Tasks up to ``n`` graph edges away from active tasks (default ``1`` edge)
-
-
-Platform Awareness
-------------------
-
-.. seealso::
-
-   - :ref:`Platforms at Cylc 8. <majorchangesplatforms>`
-   - :ref:`System admin's guide to writing platforms. <AdminGuide.PlatformConfigs>`
-
-Cylc 7 was aware of individual job hosts.
-
-.. code-block:: cylc
-
-   [runtime]
-      [[model]]
-          [[[remote]]]
-              host = hpc1.login.1  # Deprecated Cylc 8
-
-Cylc 8 is aware of host groups specified as :term:`[job] platforms <platform>`
-in the global configuration. Platform hosts share a file system and :term:`job
-runner`. If a host becomes unavailable Cylc 8 can use other hosts on the same
-platform to interact with task jobs.
-
-.. code-block:: cylc
-
-   [runtime]
-      [[model]]
-          platform = hpc1  # Cylc 8
-          # (Platform hosts and job runner defined in global config).
-      [[model_cleanup]]
-          # Platforms can have the same hosts with different job runners.
-          platform = hpc1_background
-
+Alternatively, task outputs can be marked as :term:`optional <optional output>`.
+This supports :term:`graph branching` and it allows the scheduler to
+correctly diagnose :term:`workflow completion`.
 
-.. warning::
 
-   Cylc 8 will pick a sensible platform for your Cylc 7 settings,
-   These deprecated settings will be removed at Cylc 9.
-
-
-.. _7-to-8.summary.graph_syntax:
-
-Graph Syntax
-------------
-
-Cylc 7 had unnecessarily deep nesting of graph config sections:
-
-.. code-block:: cylc
-
-   [scheduling]
-      initial cycle point = now
-      [[dependencies]]  # Deprecated Cylc 7
-          [[[R1]]]
-              graph = "prep => foo"
-          [[[R/^/P1D]]]
-              graph = "foo => bar => baz"
-
-Cylc 8 cleans this up:
-
-.. code-block:: cylc
-
-   [scheduling]
-      initial cycle point = now
-      [[graph]]  # Cylc 8
-          R1 = "prep => foo"
-          R/^/P1D = "foo => bar => baz"
-
-.. _Workflow Installation:
-
-
-Workflow Installation
----------------------
+Rose Suite-Run Migration
+------------------------
```

Comment:
I think this heading should be reverted to the original.  Not all Cylc 7 users use `rose suite-run`. Some do manual installation and/or construct suites in-place in the run directory. So the change should be: Cylc 8 supports workflow installation. Then it should be noted for Rose users that this replaces `rose suite-run` functionality.